compiler/Cargo.toml

```diff
@@ -14,15 +14,19 @@ categories = ["network-programming"]
 [features]
 default = ["protobuf-codec"]
 protobuf-codec = ["protobuf"]
+protobufv3-codec = ["protobufv3", "protobuf-codegen", "anyhow"]
```

Comment:
If the generated service code of protobuf-v3 is exact the same as protobuf-v2, then it seems no need to add a standalone features.